libs/application/templates/parental-leave/src/lib/parentalLeaveUtils.ts

```diff
@@ -465,21 +463,29 @@ export function getApplicationAnswers(answers: Application['answers']) {
 
   const selectedChild = getValueViaPath(answers, 'selectedChild') as string
 
-  const isRequestingRights = getValueViaPath(
+  const transferRights = getValueViaPath(
     answers,
-    'requestRights.isRequestingRights',
-  ) as YesOrNo
+    'transferRights',
+  ) as TransferRightsOption
+
+  const isRequestingRights =
+    transferRights === TransferRightsOption.REQUEST
+      ? YES
+      : (getValueViaPath(
+          answers,
+          'requestRights.isRequestingRights',
+        ) as YesOrNo)
```

Comment:
Is this handling the backwards compatibility issue?

Reply:
Yes